pkgs/by-name/li/libtapi/package.nix

```diff
@@ -0,0 +1,180 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchpatch,
+  cmake,
+  ninja,
+  python3,
+  zlib,
+}:
+
+let
+  # Apple’s LLVM fork uses its own versioning scheme.
```

Comment:
What's the reason for needing Apple's fork?

Reply:
Apple only supports building libtapi against their fork, which uses APIs that haven’t been upstreamed (yet).

Reply:
I added a comment with this explanation.